contrib/mesos/pkg/scheduler/executorinfo/registry.go

```diff
@@ -66,39 +64,42 @@ func NewRegistry(lookupNode node.LookupFunc, prototype *mesosproto.ExecutorInfo)
 	}, nil
 }
 
+// New creates a customized ExecutorInfo for a host
+//
+// Note: New modifies Command.Arguments and Resources and intentionally
+// does not update the executor id (although that originally depended on the
+// command arguments and the resources). But as the hostname is constant for a
```

Comment:
I originally thought it is dependent on the resources which it isn't. Comment should be updated.